app/actions.ts

```
@@ -1,321 +1,315 @@
       return {
-        transactionId: nanoid(),
-        transactionStatus: "failed",
-        recipientName: recipient.name,
-        message: `The transfer of ${amount} to ${destination} (${recipient.name}) failed. The current user has insufficient funds.`,
+        message: `Account successfully created with the following details:\n\n${JSON.stringify(
+          createdRecord.fields,
+          null,
+          2
+        )}\n\nRecord ID: ${createdRecord.id}`,
+        logs,
       };
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)